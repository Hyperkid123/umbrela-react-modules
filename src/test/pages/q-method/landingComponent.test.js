import React from 'react'
import Landing from '../../../pages/q-method/';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Q-method landing component', () => {
  it('should render correclty' , () => {
      const tree = shallow(
        <Landing/>
      ).dive();
      expect(toJson(tree)).toMatchSnapshot();
  });
});
