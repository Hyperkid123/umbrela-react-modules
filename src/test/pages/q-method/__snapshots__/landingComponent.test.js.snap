// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Q-method landing component should render correclty 1`] = `
<div
  className="sc-bwzfXH bAwxlR"
>
  <styled.div
    column={true}
  >
    <Connect(PyramidCreator) />
  </styled.div>
</div>
`;
